tools/c7n_azure/tests_azure/tests_resources/test_cosmos_db.py

```diff
@@ -13,6 +13,22 @@
 from c7n.utils import local_session
 
 
+def get_ext_ip():
+    # local external ip needs to be added to the database when recording
```

Comment:
do we really need this? generally tests work offline as well in replay mode

Reply:
I would say yes unless we want to rethink the cosmos firewall tests overall.
The big change here is 0.0.0.0 is no longer accepted via API and the firewalling must be set dynamically by the test rig.  5 of the tests rely on knowing the actual list in the portal and can no longer be a static value that matches the arm template